var/spack/repos/builtin/packages/py-pymumps/package.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPymumps(PythonPackage):
+    """Python bindings for MUMPS, a parallel sparse direct solver"""
+
+    homepage = "http://github.com/pymumps/pymumps"
+    url = "https://files.pythonhosted.org/packages/74/19/5a4405c485dc0548ad5be7536c9d19dac567c865834987ff1e998340b7ae/PyMUMPS-0.3.2.tar.gz"
+    git = "https://github.com/PyMumps/pymumps.git"
+
+    # Add a list of GitHub accounts to notify when the
+    # package is updated
+    maintainers = ['payerle']
+
+    version('0.3.2', sha256='f290ec4850098f108fb91cb9e7fa07302ebf5076e4329f8e6ea4924de8ba35df')
+
+    depends_on('py-cython', type='build')
+    depends_on('py-setuptools', type='build')
+    depends_on('mumps', type=('build', 'run'))
```

Comment:
This should at least be type=link since you are linking to the libraries and headers. build/run are only needed if you need `spec['mumps'].prefix.bin` to be in your `PATH`.